.travis/before_script.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+if [ "$TYPE" != 'style' ]; then
+  mkdir build && cd build || exit
+
+  export CMAKE_FLAGS="-DWANT_QT5=$QT5 -DUSE_WERROR=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo"
+
+  if [ -z "$TRAVIS_TAG" ]; then export CMAKE_FLAGS="$CMAKE_FLAGS -DUSE_CCACHE=ON"; fi
```

Comment:
This was likely kept a one-liner to keep the `.yml` clean reading.  Please break this out now that it's in a shell script. :)